{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "python310-Jinja2-3.1.4-1.1 on GA media",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "These are all security issues fixed in the python310-Jinja2-3.1.4-1.1 package on the GA media of openSUSE Tumbleweed.",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "openSUSE-Tumbleweed-2024-13930",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/opensuse-su-2024_13930-1.json"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2014-0012 page",
				"url": "https://www.suse.com/security/cve/CVE-2014-0012/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2016-10745 page",
				"url": "https://www.suse.com/security/cve/CVE-2016-10745/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2019-10906 page",
				"url": "https://www.suse.com/security/cve/CVE-2019-10906/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2020-28493 page",
				"url": "https://www.suse.com/security/cve/CVE-2020-28493/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2024-34064 page",
				"url": "https://www.suse.com/security/cve/CVE-2024-34064/"
			}
		],
		"title": "python310-Jinja2-3.1.4-1.1 on GA media",
		"tracking": {
			"current_release_date": "2024-06-15T00:00:00Z",
			"generator": {
				"date": "2024-06-15T00:00:00Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "openSUSE-SU-2024:13930-1",
			"initial_release_date": "2024-06-15T00:00:00Z",
			"revision_history": [
				{
					"date": "2024-06-15T00:00:00Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Jinja2-3.1.4-1.1.aarch64",
								"product": {
									"name": "python310-Jinja2-3.1.4-1.1.aarch64",
									"product_id": "python310-Jinja2-3.1.4-1.1.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Jinja2-3.1.4-1.1.aarch64",
								"product": {
									"name": "python311-Jinja2-3.1.4-1.1.aarch64",
									"product_id": "python311-Jinja2-3.1.4-1.1.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Jinja2-3.1.4-1.1.aarch64",
								"product": {
									"name": "python312-Jinja2-3.1.4-1.1.aarch64",
									"product_id": "python312-Jinja2-3.1.4-1.1.aarch64"
								}
							}
						],
						"category": "architecture",
						"name": "aarch64"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Jinja2-3.1.4-1.1.ppc64le",
								"product": {
									"name": "python310-Jinja2-3.1.4-1.1.ppc64le",
									"product_id": "python310-Jinja2-3.1.4-1.1.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Jinja2-3.1.4-1.1.ppc64le",
								"product": {
									"name": "python311-Jinja2-3.1.4-1.1.ppc64le",
									"product_id": "python311-Jinja2-3.1.4-1.1.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Jinja2-3.1.4-1.1.ppc64le",
								"product": {
									"name": "python312-Jinja2-3.1.4-1.1.ppc64le",
									"product_id": "python312-Jinja2-3.1.4-1.1.ppc64le"
								}
							}
						],
						"category": "architecture",
						"name": "ppc64le"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Jinja2-3.1.4-1.1.s390x",
								"product": {
									"name": "python310-Jinja2-3.1.4-1.1.s390x",
									"product_id": "python310-Jinja2-3.1.4-1.1.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Jinja2-3.1.4-1.1.s390x",
								"product": {
									"name": "python311-Jinja2-3.1.4-1.1.s390x",
									"product_id": "python311-Jinja2-3.1.4-1.1.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Jinja2-3.1.4-1.1.s390x",
								"product": {
									"name": "python312-Jinja2-3.1.4-1.1.s390x",
									"product_id": "python312-Jinja2-3.1.4-1.1.s390x"
								}
							}
						],
						"category": "architecture",
						"name": "s390x"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Jinja2-3.1.4-1.1.x86_64",
								"product": {
									"name": "python310-Jinja2-3.1.4-1.1.x86_64",
									"product_id": "python310-Jinja2-3.1.4-1.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Jinja2-3.1.4-1.1.x86_64",
								"product": {
									"name": "python311-Jinja2-3.1.4-1.1.x86_64",
									"product_id": "python311-Jinja2-3.1.4-1.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Jinja2-3.1.4-1.1.x86_64",
								"product": {
									"name": "python312-Jinja2-3.1.4-1.1.x86_64",
									"product_id": "python312-Jinja2-3.1.4-1.1.x86_64"
								}
							}
						],
						"category": "architecture",
						"name": "x86_64"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Jinja2-3.1.4-1.1.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64"
				},
				"product_reference": "python310-Jinja2-3.1.4-1.1.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Jinja2-3.1.4-1.1.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le"
				},
				"product_reference": "python310-Jinja2-3.1.4-1.1.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Jinja2-3.1.4-1.1.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x"
				},
				"product_reference": "python310-Jinja2-3.1.4-1.1.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Jinja2-3.1.4-1.1.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64"
				},
				"product_reference": "python310-Jinja2-3.1.4-1.1.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Jinja2-3.1.4-1.1.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64"
				},
				"product_reference": "python311-Jinja2-3.1.4-1.1.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Jinja2-3.1.4-1.1.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le"
				},
				"product_reference": "python311-Jinja2-3.1.4-1.1.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Jinja2-3.1.4-1.1.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x"
				},
				"product_reference": "python311-Jinja2-3.1.4-1.1.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Jinja2-3.1.4-1.1.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64"
				},
				"product_reference": "python311-Jinja2-3.1.4-1.1.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Jinja2-3.1.4-1.1.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64"
				},
				"product_reference": "python312-Jinja2-3.1.4-1.1.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Jinja2-3.1.4-1.1.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le"
				},
				"product_reference": "python312-Jinja2-3.1.4-1.1.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Jinja2-3.1.4-1.1.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x"
				},
				"product_reference": "python312-Jinja2-3.1.4-1.1.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Jinja2-3.1.4-1.1.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
				},
				"product_reference": "python312-Jinja2-3.1.4-1.1.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2014-0012",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2014-0012"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "FileSystemBytecodeCache in Jinja2 2.7.2 does not properly create temporary directories, which allows local users to gain privileges by pre-creating a temporary directory with a user's uid. NOTE: this vulnerability exists because of an incomplete fix for CVE-2014-1402.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2014-0012",
					"url": "https://www.suse.com/security/cve/CVE-2014-0012"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 858239 for CVE-2014-0012",
					"url": "https://bugzilla.suse.com/858239"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2014-0012"
		},
		{
			"cve": "CVE-2016-10745",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2016-10745"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In Pallets Jinja before 2.8.1, str.format allows a sandbox escape.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2016-10745",
					"url": "https://www.suse.com/security/cve/CVE-2016-10745"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1132174 for CVE-2016-10745",
					"url": "https://bugzilla.suse.com/1132174"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 8.7,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:C/C:H/I:H/A:L",
						"version": "3.0"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2016-10745"
		},
		{
			"cve": "CVE-2019-10906",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2019-10906"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "In Pallets Jinja before 2.10.1, str.format_map allows a sandbox escape.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2019-10906",
					"url": "https://www.suse.com/security/cve/CVE-2019-10906"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1132323 for CVE-2019-10906",
					"url": "https://bugzilla.suse.com/1132323"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 8.1,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:L/AC:H/PR:N/UI:N/S:C/C:H/I:H/A:L",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2019-10906"
		},
		{
			"cve": "CVE-2020-28493",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2020-28493"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "This affects the package jinja2 from 0.0.0 and before 2.11.3. The ReDoS vulnerability is mainly due to the `_punctuation_re regex` operator and its use of multiple wildcards. The last wildcard is the most exploitable as it searches for trailing punctuation. This issue can be mitigated by Markdown to format user content instead of the urlize filter, or by implementing request timeouts and limiting process memory.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2020-28493",
					"url": "https://www.suse.com/security/cve/CVE-2020-28493"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1181944 for CVE-2020-28493",
					"url": "https://bugzilla.suse.com/1181944"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "important"
				}
			],
			"title": "CVE-2020-28493"
		},
		{
			"cve": "CVE-2024-34064",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2024-34064"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Jinja is an extensible templating engine. The `xmlattr` filter in affected versions of Jinja accepts keys containing non-attribute characters. XML/HTML attributes cannot contain spaces, `/`, `>`, or `=`, as each would then be interpreted as starting a separate attribute. If an application accepts keys (as opposed to only values) as user input, and renders these in pages that other users see as well, an attacker could use this to inject other attributes and perform XSS. The fix for CVE-2024-22195 only addressed spaces but not other characters. Accepting keys as user input is now explicitly considered an unintended use case of the `xmlattr` filter, and code that does so without otherwise validating the input should be flagged as insecure, regardless of Jinja version. Accepting _values_ as user input continues to be safe. This vulnerability is fixed in 3.1.4.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
					"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2024-34064",
					"url": "https://www.suse.com/security/cve/CVE-2024-34064"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1223980 for CVE-2024-34064",
					"url": "https://bugzilla.suse.com/1223980"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 6.1,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python310-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python311-Jinja2-3.1.4-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.s390x",
						"openSUSE Tumbleweed:python312-Jinja2-3.1.4-1.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-06-15T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2024-34064"
		}
	]
}
